types/serviceensure.pp

```diff
@@ -0,0 +1 @@
+type Stdlib::Serviceensure = Enum['stopped', 'running']
```

Comment:
should this not be 
```ruby
type Stdlib::Serviceensure = Varient[Boolean,Enum['stopped', 'running']]
```

Reply:
IMO, using `ensure => true` or `ensure => false` on a service is an anti-pattern.  It works but it's just not as clear to new users what is going on.  As a convention I don't support it.

Reply:
I tend to agree with you however if this is going into stdlib then i think it should conform to the documentation 
https://puppet.com/docs/puppet/5.3/type.html#service-attribute-ensure